Introduce new page for saml just in time jit provisioning (#1114)
clerk/clerk-docs: docs/authentication/saml/jit-provisioning.mdx

> ---

Reviewer: Does this require its own guide or could this be added as a section to /saml/overview right above the "[Sync user attributes during sign in](https://clerk.com/docs/pr/1114/authentication/saml/overview#sync-user-attributes-during-sign-in)" section ?

I feel like 1. It's basically a concept that is inextricably tied to SAML and the explanation of how we do SAML so it'd make sense to be in the overview, and 2. it'd be more easily found there

Author: That's not a bad idea, @chanioxaris what do you think?

Reviewer 2: Even if this page doesn't have much content for the time being, JIT provisioning is one of the important features that comes with SAML that existing and potential customers are greatly interested in. So i would say it deserves it's own page to be explicit and mostly about be SEO friendly and easily discoverable.

We should also move to this new page the [Sync user attributes during sign in](https://clerk.com/docs/authentication/saml/overview#sync-user-attributes-during-sign-in) section from the overview page, as describes how JIT provisioning works during sign in

Author: Got it, thanks for the input 👍

> description: Learn about Clerk's support for Just-In-Time account provisioning with all supported SAML providers.

Reviewer 2: I would say let's stick with the term `Just-in-Time (JIT) Provisioning` for the rest of the page, which is the most common one

```suggestion
description: Learn about Clerk's support for Just-in-Time (JIT) Provisioning during SAML SSO.
```

---

> # JIT account provisioning with SAML

Reviewer 2: ```suggestion
# Just-in-Time (JIT) Provisioning during SAML SSO
```

> Just-in-Time (JIT) account provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand when they sign-up to an app with SAML using a company email address.

Reviewer 2: ```suggestion
Just-in-Time (JIT) Provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand during the first time they authenticate via SAML SSO in your application.
```

> Using JIT account provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done. Consider using this feature if you don't need to control who accesses your Clerk application on a case-by-case basis.

Reviewer 2: This is supported by default and customers don't have a way to opt-out for the time being
```suggestion
Using Just-in-Time (JIT) Provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done.
```

> Clerk supports JIT account provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](/docs/authentication/saml/overview#sync-user-attributes-during-sign-in)** setting enabled, which is enabled by default.

Reviewer 2: The Sync user attributes during Sign in setting, only controls the behavior during sign in. We should move the relevant section from the SAML overview page to this page

```suggestion
Clerk supports Just-in-Time (JIT) Provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box.
```

> Clerk supports JIT account provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](/docs/authentication/saml/overview#sync-user-attributes-during-sign-in)** setting enabled, which is all Clerk apps by default.

Reviewer: ```suggestion
Clerk supports JIT account provisioning for all [supported SAML providers](/docs/pr/1038/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](/docs/authentication/saml/overview#sync-user-attributes-during-sign-in)** setting enabled, which is enabled by default.
```

> The following is a list of guides for enabling JIT account provisioning with some popular SAML providers:

Reviewer 2: I would say avoid adding links here about the supported SAML providers. I would bet, when we add a new SAML provider, we will forget to update this page. If we want to include them, we should do it as part of the dedicated provider guides

---
description: Learn about Clerk's support for Just-in-Time (JIT) Provisioning during SAML SSO.
---

# Just-in-Time (JIT) Provisioning during SAML SSO

Just-in-Time (JIT) Provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand during the first time they authenticate via SAML SSO in your application.

Using JIT Provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done.

> Clerk supports JIT account provisioning for all [supported SAML providers](/docs/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](#sync-user-attributes-during-sign-in)** setting enabled, which is enabled by default.

Reviewer 2: Customers can only control if they want the attributes to be synced in each sign in using the  `Sync user attributes during Sign in` option. They can't control the behavior during the initial sign-up, so it's better to drop this line as we have a dedicated section below for the sign in setting

```suggestion
Clerk supports JIT account provisioning for all [supported SAML providers](/docs/authentication/saml/overview) out of the box.
```
